x-pack/plugins/ml/public/application/management/jobs_list/_index.scss

```diff
@@ -1 +1,4 @@
-@import 'components/index';
+// Kibana management page ML section
+#kibanaManagementMLSection {
+  @import 'components/index';
+}
```

Comment:
Can we switch this so it uses a three-letter prefix on the classes, instead of nesting within an id?

Reply:
Hi @mdefazio! Sorry, I already merged this one, but the point of this PR was to rearrange some imports to reduce the bundle size we deliver during kibana loading. We definitely need to revisit our SCSS files, I believe it deserves a dedicated PR.